scipy/integrate/_quadrature.py

```diff
@@ -1035,3 +1039,242 @@ def newton_cotes(rn, equal=0):
+    i.i.d. random samples from the population of integral estimates.
+    The sample mean :math:`m` of these integral estimates is an
+    unbiased estimator of the true value of the integral, and the standard
+    error of the mean :math:`s` of these estimates may be used to generate
+    confidence intervals using the t distribution with ``n_estimates - 1``
+    degrees of freedom.
+
+
+    Examples
+    --------
+    QMC quadrature is particularly useful for computing integrals in higher
+    dimensions; an example integrand is the probability density function
+    of a multivariate normal distribution.
+
+    >>> import numpy as np
+    >>> from scipy import stats
+    >>> dim = 8
+    >>> mean = np.zeros(dim)
+    >>> cov = np.eye(dim)
+    >>> def func(*z):
+    ...     z = np.asarray(z).T
+    ...     return stats.multivariate_normal.pdf(z, mean, cov)
+
+    To compute the integral over the unit hypercube:
+
+    >>> from scipy.integrate import qmc_quad
+    >>> lb = np.zeros(dim)
+    >>> ub = np.ones(dim)
+    >>> ranges = np.asarray([lb, ub]).T
+    >>> rng = np.random.default_rng()
+    >>> qrng = stats.qmc.Halton(d=dim, seed=rng)
+    >>> n_estimates = 8
+    >>> res = qmc_quad(func, ranges, n_estimates=n_estimates, qrng=qrng)
+    >>> res.integral, res.standard_error
+    (0.00018443157359780213, 8.95876227122206e-08)
+
+    A two-sided, 99% confidence interval for the integral may be estimated
+    as:
+
+    >>> t = stats.t(df=n_estimates-1, loc=res.integral,
+    ...             scale=res.standard_error)
+    >>> t.interval(0.99)
+    (0.00018411806320847138, 0.0001847450839871329)
+
+    Indeed, the value reported by `scipy.stats.multivariate_normal` is
+    within this range.
+
+    >>> stats.multivariate_normal.cdf(ub, mean, cov, lower_limit=lb)
+    0.00018430867675187443  # may vary
+
+    """
+    args = _qmc_quad_iv(func, ranges, n_points, n_estimates, qrng, log, args)
+    func, ranges, n_points, n_estimates, qrng, rng, log, args, stats = args
+
+    lb, ub = ranges.T
+
+    # The sign of the integral depends on the order of the limits. Fix this by
+    # ensuring that lower bounds are indeed lower and setting sign of resulting
+    # integral manually
+    i_swap = ub < lb
+    sign = (-1)**(i_swap.sum(axis=-1))  # odd # of swaps -> negative
+    lb[i_swap], ub[i_swap] = ub[i_swap], lb[i_swap]
+
+    A = np.prod(ub - lb)
+    dA = A / n_points
+
+    estimates = np.zeros(n_estimates)
+    for i in range(n_estimates):
+        # Generate integral estimate
+        sample = qrng.random(n_points)
+        x = stats.qmc.scale(sample, lb, ub)
+        integrands = func(*x.T, *args)
+        if log:
+            estimate = logsumexp(integrands) + np.log(dA)
+        else:
+            estimate = np.sum(integrands * dA)
+        estimates[i] = estimate
+
+        # Get a new, independently-scrambled QRNG for next time
+        if hasattr(qrng._init, 'scramble'):
```

Comment:
Did you mean this? Why not just add `scramble` to the `_init`s when they are created?

```suggestion
        if hasattr(qrng, 'scramble'):
```

Reply:
> Did you mean this? 

Yes, this would work, too. I didn't notice that they had `scramble` as an attribute.

> Why not just add `scramble` to the `_inits` when they are created?

I did, but I set it to whatever the user passed in (`"scramble": scramble`), whereas here I want it to be `True` regardless of what the user passed in. 

I suppose I was thinking that `_init` should faithfully record what the user passed in. But if that were my philosophy, it should have included `seed`, too, and it doesn't. `_init` is new in this PR, and not used elsewhere, so I might as well make it suit my needs here.

I'll change `_init` to always have `"scramble": True` so we can avoid this logic altogether.